src/consts/consts.ts

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line max-len
+export const API_URL = 'https://mate-academy.github.io/react_people-table/api/people.json';
```

Comment:
variable like `API_URL`, `API_KEY` better to store in [env](https://mate.academy/learn/react/deployment?section=video&videoId=781) file

Reply:
Idk why but I can't use dotenv properly now(it was in previous projects as well). Maybe I need to configure something on my pc or IDE to fix it so I can't provide that functionality in that task in that moment.